docs/logical_data_model.md

```diff
@@ -16,10 +16,51 @@ class User {
   hsesUserId : string
   name : string
   phoneNumber : string
-  email : string
+  * email : string
```

Comment:
Did you mean to update the image above?

Reply:
Yes, updating of the image was intended. Do you see anything wrong with the image?

Reply:
Google isn't letting me upload a screenshot for some reason, but I still see the old image. You'll need to update the url on line 4 to use the new image.

Reply:
Thanks for checking this. Yes, line 4 was updated, but the file was not committed :-)
It is now.